common/plugins/eu.esdihumboldt.hale.common.schema/src/eu/esdihumboldt/hale/common/schema/io/impl/AbstractSchemaReader.java

```diff
@@ -32,6 +32,8 @@
 
 	private SchemaSpaceID schemaSpace;
 
+	private Boolean isSchemaNameQuoted;
```

Comment:
This configuration for adapting the behavior of the `JDBCSchemaReader` should be done in that class, and not in the `AbstractSchemaReader`. The `AbstractSchemaReader` has no association with JDBC or databases in general and is used as base for any schema reader class.

Also, please use a `boolean` with the default value representing the current behavior. Please think about the choice of name as well, because the quoting not only affects the schema name, but at least also table names. Maybe something more generic like `useQuotes` or similar.